File: Code Boiler/golang/Wallet.go
package main

import "fmt"

type Wallet struct {
	balance int
}

func (w *Wallet) ViewBalance() {
	fmt.Println("Your balance is: ", w.balance)
}

func (w *Wallet) Deposit(amount int) {
	w.balance += amount
	fmt.Printf("%d has been added to your account. Your new balance is: %d\n", amount, w.balance)
}

func main() {
	wallet := Wallet{}
	wallet.ViewBalance()
	wallet.Deposit(50)
	wallet.ViewBalance()
}

File: Code Boiler/Java/Wallet.java
package signup.Java;

public class Wallet {
    private int balance = 0; 

    public void viewBalance() {
        System.out.println("Your balance is " + balance); 
    }


    public void deposit(int amount) {
        balance += amount; 
        System.out.println(amount + " has been added to your account. Your new balance is: " + balance);
    }

    public static void main(String[] args) {
        Wallet wallet = new Wallet(); 
        wallet.viewBalance(); 
        wallet.deposit(50); 
        wallet.viewBalance(); 
    }
    
}
